### ISSUE TITLE
Different code generated on MacOS and Ubuntu

### ISSUE THREAD

**Issue by kassim** [localizely/flutter-intl-plugin-sample-app#4]
When you have 3 arb files with locales 'en' 'en_US' 'fr_FR' and you run the command `flutter pub run intl_utils:generate`

On MacOS the l10n.dart file generated has
```
List<Locale> get supportedLocales {
 return const <Locale>[
  Locale.fromSubtags(languageCode: 'en'), Locale.fromSubtags(languageCode: 'fr', countryCode: 'FR'), Locale.fromSubtags(languageCode: 'en', countryCode: 'US'),
 ];
}
```

on Ubuntu the ordering is
```
Locale.fromSubtags(languageCode: 'en'), Locale.fromSubtags(languageCode: 'en', countryCode: 'US'), Locale.fromSubtags(languageCode: 'fr', countryCode: 'FR'),
```

**Comment by aleksakrstic** [comment]
Hi @kassim 
Yes, this is known issue among different Operating Systems.
Will be fixed soon with https://github.com/localizely/flutter-intl-vscode/issues/21

**Comment by aleksakrstic** [comment]
@kassim 
This should be fixed with new IDE plugin version `1.5.0` and `intl_utils` version `1.3.0`.
